collective/elasticsearch/tests/test_search.py

```diff
@@ -121,43 +122,42 @@ def test_path_index_query(self):
 
         self.commit()
         self.es.connection.indices.flush()
+        time.sleep(1)
 
         self.assertEqual(
-            len(self.catalog(path={'depth': 0, 'query': '/plone/folder0'},
```

Comment:
`len` should work on catalog result.

Reply:
In this case, works on "len" cause the result is "integer"

Reply:
It must be possible to use `len`, regardless of how the ES returns the result.

Reply:
Yes, if you filter the result, the new version of ES bring the amount e how the amount is exact.

Reply:
Please change the implementation not the tests